packages/@aws-cdk/core/lib/stack-synthesizers/default-synthesizer.ts

```diff
@@ -266,6 +266,23 @@ export class DefaultStackSynthesizer implements IStackSynthesizer {
     }, [artifactId]);
   }
 
+  /**
+   * Create a copy of this synthesizer,
+   * with all of the fields the same,
+   * but not bound to any stack yet.
+   *
+   * @experimental
+   */
+  public bootstraplessCopy(): IStackSynthesizer {
+    return new BootstraplessSynthesizer({
+      fileAssetsBucketName: this.bucketName,
```

Comment:
If it's bootstrapless it can't need this.

Reply:
Removed.